components/common/TagGroup.js

```diff
@@ -1,14 +1,41 @@
 import React from 'react'
-import { BlueTag } from './Tags'
+import { BlueTag, SelectedTag } from './Tags'
 
 export default (props) => {
-  const { tags } = props
+  let { tags, selectedTags, updateTag } = props
   if (!tags || !tags.length) return null
+  selectedTags = selectedTags || []
+  // TODO: Use same tag format between DropdownFilter and TagGroup
   return (
-    tags.map(({ id, name }) => (
-      <BlueTag key={id} className="tag is-rounded has-text-white">
-        {name}
+    tags.map(tag => {
+      const matchedTag = selectedTags.find(({ value }) => value === tag.id)
+      if (matchedTag) {
+        return <SelectedTag
+          key={tag.value}
+          className="tag is-rounded has-text-white"
+          onClick={(e) => {
+            e.stopPropagation()
```

Comment:
Maybe put a comment as to why we need to stop propagation

Reply:
done